fem/intrules.cpp

```diff
@@ -1114,6 +1129,7 @@ IntegrationRule *IntegrationRules::PointIntegrationRule(int Order)
    IntegrationRule *ir = new IntegrationRule(1);
    ir->IntPoint(0).x = .0;
    ir->IntPoint(0).weight = 1.;
+   ir->SetOrder(1);
```

Comment:
Isn't the order of this rule in some sense infinite?

Reply:
♾️ 😮

Reply:
Seems fine. I agree, it's probably never used. It is an interesting corner case, though.